cmd/apiserver/app/server/util.go

```diff
@@ -91,7 +91,13 @@ func buildGenericConfig(s *ServiceCatalogServerOptions) (*genericapiserver.Recom
 		genericConfig.Authorization.Authorizer = authorizerfactory.NewAlwaysAllowAuthorizer()
 	}
 
-	if err := s.AuditOptions.ApplyTo(&genericConfig.Config); err != nil {
+	if err := s.AuditOptions.ApplyTo(
+		&genericConfig.Config,
+		genericConfig.ClientConfig,
+		genericConfig.SharedInformerFactory,
+		// TODO where to get namespace? Downward api?
```

Comment:
Not sure what to do here

Reply:
https://github.com/kubernetes-sigs/controller-runtime/blob/50b6bdd0660bf82fdedc64bae06e5a2e878685bc/pkg/leaderelection/leader_election.go#L93